beetsplug/deezer.py

```diff
@@ -108,8 +116,14 @@ def album_for_id(self, album_id):
                 "by {} API: '{}'".format(self.data_source, release_date)
             )
 
-        tracks_obj = requests.get(self.album_url + deezer_id + "/tracks").json()
-        tracks_data = tracks_obj["data"]
+        try:
+            tracks_obj = requests.get(
+                self.album_url + deezer_id + "/tracks", timeout=10
+            ).json()
+            tracks_data = tracks_obj["data"]
+        except requests.exceptions.ReadTimeout:
```

Comment:
Any particular reason to use `ReadTimeout` here instead of the more general `ConnectionError`?